be/database/migrations/2023_07_21_023821_create_reservation_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('reservation', function (Blueprint $table) {
+            $table->id();
+            $table->date('check_in');
+            $table->date('check_out');
+            $table->float('total_price');
+            $table->boolean('arrived')->default(false);
+            $table->boolean('is_approve')->default(false);
+            $table->boolean('is_cancel')->default(false);
+            $table->foreignId('client_id')->constrained('client')->cascadeOnDelete();
+            $table->foreignId('room_id')->constrained('room')->cascadeOnDelete();
+            $table->foreignId('host_id')->constrained('host')->cascadeOnDelete();
+
+            $table->timestamps();
```

Comment:
```suggestion
        Schema::create('reservation', function (Blueprint $table) {
            $table->id();
            $table->foreignId('client_id')->constrained('client')->cascadeOnDelete();
            $table->foreignId('room_id')->constrained('room')->cascadeOnDelete();
            $table->foreignId('host_id')->constrained('host')->cascadeOnDelete();
            $table->date('check_in');
            $table->date('check_out');
            $table->float('total_price');
            $table->boolean('arrived')->default(false);
            $table->boolean('is_approve')->default(false);
            $table->boolean('is_cancel')->default(false);
            $table->timestamps();
```
Foreign keys must be prioritized before the other columns

Reply:
understood sir